build.sbt

```diff
@@ -216,9 +216,9 @@ lazy val microsite = project.module
     micrositeDocumentationLabelDescription := "Scaladoc",
     micrositeBaseUrl := "/scalaz-zio",
     micrositePalette := Map(
-      "brand-primary"   -> "#ED2124",
-      "brand-secondary" -> "#251605",
-      "brand-tertiary"  -> "#491119",
+      "brand-primary"   -> "#86201e",
```

Comment:
I've seen the new logo somewhere and it would be nice if the colour palette took hints of it and provided similar as well as contrasting colours for callouts etc.